metabase/metabase
# Fix cutting of data points in line/area/bar charts

Fixes https://github.com/metabase/metabase/issues/17923

How to test:
- Create a line chart, Orders, grouped by Created At (year) -> Count
- Enable data points
- It should display the data point on the top of the chart without cutting it off
- There are also CY tests for this

<img width="1126" alt="Screen Shot 2021-09-29 at 8 14 39 pm" src="https://user-images.githubusercontent.com/8542534/135317391-ffd54982-4723-4930-9f78-6178e08ddb8c.png">
